spec/beaker/dsl/install_utils/foss_utils_spec.rb

```diff
@@ -1131,6 +1139,110 @@ def test_fetch_http_file_no_ending_slash(platform)
       subject.remove_puppet_on( aixhost )
     end
 
+    it 'uninstalls packages on el' do
```

Comment:
@johnduarte can have these uninstall tests run through an array of hosts, rather than copying the test for each platform it now supports?